libs/blocks/caas-config/caas-config.css

```diff
@@ -36,3 +36,72 @@ dd.content .sort-options > .field {
   top: 12px;
 }
 
+.config-panel dd.content.is-expanded {
+  max-height: 6000px;
```

Comment:
6000px max-height looks a little weird, is it intended?

Reply:
Since we don't know how many groups of filters and filters will be added I hope 6000 pixels is enough. the original 1800pixes was not enough.